torch/utils/data/dataloader.py

```diff
@@ -192,15 +199,13 @@ def __init__(self, loader):
             self.rcvd_idx = 0
             self.reorder_dict = {}
 
+            base_seed = torch.LongTensor(1).random_()[0]
```

Comment:
Changed to `worker_seed <- main_proc_generated_long + worker_id` so that they can be unique and testable.